lib/pbench/cli/server/user_update.py

```diff
@@ -0,0 +1,107 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-user-update"
+
+
+class UserUpdate:
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        config = PbenchServerConfig(self.context.config)
+
+        logger = get_pbench_logger(_NAME_, config)
+
+        # We're going to need the Postgres DB to track dataset state, so setup
+        # DB access.
+        Database.init_db(config, logger)
+
+        # Check if the user has the right privileges
+        if user is None or not user.is_admin():
+            raise RuntimeError(f"Not authorized to update user {self.context.username}")
+
+        dict_to_update = {}
+        if self.context.username:
+            dict_to_update["username"] = self.context.username
```

Comment:
We're editing username, but we don't _list_ username; we should at least be consistent in whether we're pretending that username doesn't exist.